cmd/node/main.go

```diff
@@ -1530,9 +1530,27 @@ func applyCompatibleConfigs(isInImportMode bool, importDbNoSigCheckFlag bool, lo
 		config.Heartbeat.DurationToConsiderUnresponsiveInSec = math.MaxInt32
 		config.Heartbeat.MinTimeToWaitBetweenBroadcastsInSec = math.MaxInt32 - 2
 		config.Heartbeat.MaxTimeToWaitBetweenBroadcastsInSec = math.MaxInt32 - 1
+
+		changeStorageConfigs(config)
 	}
 }
 
+func changeStorageConfigs(config *config.Config) {
```

Comment:
`alterStorageConfigsForDBImport` ?
also, you could send value instead of pointer

Reply:
I will change the name but I need to keep the pointer to be modified by the function

Reply:
done